integration-tests/react-percy-storybook/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
```

Comment:
It's important that each of the integration test packages be marked as private so Lerna doesn't try to publish them.